app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -62,6 +62,7 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
     private lateinit var setupDevice: Preference
     private lateinit var settingsVoice: Preference
     private var flag = true
+    private val packageName = "ai.susi";
```

Comment:
BuildConfig.APPLICATION_ID

Reply:
@iamareebjamal For fdroid flavor it becomes ai.susi.fdroid, which can't be opened on play store

Reply:
That's why I have hardcoded it 👍

Reply:
Did you try getting it this way?

Reply:
@iamareebjamal Yes, I did! check the build it has ```applicationIdSuffix ".fdroid"```

Reply:
That's fine, I know that. I wanted to know what `BuildConfig.APPLICATION_ID` returned

Reply:
@iamareebjamal For fdroid version, it returned ai.susi.fdroid while for playstore version it returned ai.susi